srtp.go

```diff
@@ -54,45 +54,60 @@ func (c *Context) DecryptRTP(dst, encrypted []byte, header *rtp.Header) ([]byte,
 	return c.decryptRTP(dst, encrypted, header)
 }
 
-// EncryptRTP encrypts a plaintext RTP packet, writing to the dst buffer provided.
-// If the dst buffer does not have the capacity to hold `len(plaintext) + 10` bytes, a new one will be allocated.
-// If a rtp.Header is provided, it will be Unmarshaled using the plaintext.
-func (c *Context) EncryptRTP(dst []byte, plaintext []byte, header *rtp.Header) ([]byte, error) {
-	// TODO(@lcurley) Potentially accept a *rtp.Packet to avoid this Unmarshal
-	if header == nil {
-		header = &rtp.Header{}
-	}
+// EncryptRTP marshals and encrypts an RTP packet, writing to the dst buffer provided.
+// If the dst buffer does not have the capacity to hold `len(plaintext) + 10` bytes, a new one will be allocated and returned.
+// If the header is provided, it will be unmarshaled using the plaintext.
+func (c *Context) EncryptRTP(dst []byte, plaintext []byte, header *rtp.Header) (ciphertext []byte, err error) {
```

Comment:
I would like to replace with API with `EncryptRTP(dst []byte, packet *rtp.Packet) (ciphertext []byte, err error)` as seen below. It's not critical so I didn't make the change.